google_permissions/examples/basic_with_addon/main.tf

```
@@ -0,0 +1,14 @@
+module "permissions" {
+  source = "../../../google_permissions"
+  # it is assumed that you loaded and have available a local.project
+  google_folder_id  = local.project.folder.id
+  google_prod_id    = local.project["prod"].id
+  google_nonprod_id = local.project["nonprod"].id
+  admin_members     = ["workgroup:my-project/admins"]
+  developer_members = ["workgroup:my-project/developers"]
+  other_permissions = [
```

```suggestion
  other_permissions = [
```

Let's discuss how we want to expose this to users later today. I feel like we should use the actual role names instead of these stubbed names to reduce confusion. Additionally, we should should have some feedback (maybe via terraform validation conditions) for non supported roles, document supported roles, or both.